apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -153,8 +161,33 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
                     deletedAt: null,
                 })
 
+                // cache billing receipt for calculation of is payable field
+                const minPeriod = min(receiptsForConsumer.map(item => item.period))
```

Comment:
minPeriod can be a year ago for some receipts and it will cause to load lots of receipts for other serviceConsumers